src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -307,14 +316,10 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
             innerText: {
                 // NOTE: http://caniuse.com/#search=Node.innerText
-                condition: el => typeof el.innerText === 'string' &&
-                                 (domUtils.isScriptElement(el) || domUtils.isStyleElement(el)),
+                condition: el => typeof el.innerText === 'string' && domUtils.isElementNode(el),
 
                 get: el => {
-                    if (domUtils.isScriptElement(el))
-                        return removeProcessingHeader(el.innerText);
-
-                    return styleProcessor.cleanUp(el.innerText, urlUtils.parseProxyUrl);
+                    return el.innerText ? PropertyAccessorsInstrumentation.cleanUpNodeText(el.innerText) : el.innerText;
```

Comment:
Add code
```js
if (!text)
   return text;
```

for `cleanUpNodeText` function

 and you can simplify this to 
```js
get: el =>PropertyAccessorsInstrumentation.cleanUpNodeText(el.innerText)
```
and same cases below